tools/cli/internal/changelog/changelog.go

```diff
@@ -173,6 +174,11 @@ func NewEntries(basePath, revisionPath, exceptionFilePath string) ([]*Entry, err
 	}
 
 	for _, version := range changelog.RevisionMetadata.Versions {
+		// Skip preview versions
+		if apiversion.IsPreviewSabilityLevel(version) {
```

Comment:
Could we fix function name while we're here? (these functions: https://github.com/mongodb/openapi/blob/main/tools/cli/internal/apiversion/stabilitylevel.go#L31-L47)
```suggestion
		if apiversion.IsPreviewStabilityLevel(version) {
```

Reply:
yea! thank you!